readme.md

```diff
@@ -14,6 +14,12 @@ This package works with both sandboxed and non-sandboxed apps and it's App Store
 
 ## Install
 
+#### Swift Package Manager
+
+```
+.package(url: "https://github.com/sindresorhus/LaunchAtLogin", from: "3.0.2")
+```
+
 #### Carthage
 
 ```
```

Comment:
This needs to mention that using Carthage is not recommended at that support for it will be removed at some point in the future.